ctl/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "ctl"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
rm this line.

Reply:
This lines seems to be added by cargo automatically 😢 , I have not noticed this changed yet